org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/validation/ValidatingEditorCallback.java

```diff
@@ -54,9 +54,11 @@ public void afterSave(XtextEditor editor) {
 	private ValidationJob newValidationJob(XtextEditor editor) {
 		IValidationIssueProcessor issueProcessor;
 		if (editor.getResource() == null) {
-			issueProcessor = new AnnotationIssueProcessor(editor.getDocument(), editor.getInternalSourceViewer().getAnnotationModel(), issueResolutionProvider);
+			issueProcessor = new AnnotationIssueProcessor(editor.getDocument(), editor.getInternalSourceViewer().getAnnotationModel(),
```

Comment:
Should we expose getAnnotationModel on the editor?

Reply:
Well, this exposure is not new but is already used in AnnotationIssueProcessor. Probably it would be more nice to not expose the model but that would be quite a lot of refactoring work.